src/huak/ops/run.rs

```diff
@@ -0,0 +1,14 @@
+use crate::{
+    errors::{HuakError, HuakResult},
+    project::Project,
+};
+
+pub fn run_command(project: &Project, command: &str) -> HuakResult<()> {
+    let venv = match project.venv() {
+        Some(v) => v,
+        None => return Err(HuakError::VenvNotFound),
+    };
+
+    let arglist: Vec<&str> = command.split_whitespace().collect();
+    venv.exec_module(arglist[0], &arglist[1..], &project.root)
```

Comment:
hmm. this is not really what I envisioned. maybe i need to update the issue. so that's probably my bad.

how I see this working is similar to `poetry run` where you can run more than just modules installed to the venv. it's more of a way to pass commands into the context of the venv.

for example you can run `make` commands using `poetry run` not that it'd always make sense to.

Reply:
I completely missed the github issue and the discussion there regarding the implementation of the run command. I came to this via searching for unimplemented.

I will try to write a nicer implementation exposing the venv into the shell environment before running any arbitrary command.

Reply:
Take a look at the implementation of the `activate` command. `run` should be pretty much the same, but instead of spawning pseudo-terminal it would just execute the given command in that shell.